src/openrct2/localisation/Localisation.cpp

```diff
@@ -903,13 +903,14 @@ static void format_velocity(char** dest, size_t* size, uint16_t value)
     switch (gConfigGeneral.measurement_format)
     {
         default:
```

Comment:
Shouldn't you erase this `default` given that you handled all cases?

Reply:
So I kept it in (here and elsewhere) as a fallback in case we add another value to the enum and forget to update the switches.  That said, it seems pretty unlikely we'd do that, and these methods wouldn't be doing the right thing anyway in that case.

Happy to remove them if you'd prefer that, let me know!

Reply:
You pretty much explained why I'd like not to have the default labels hahahah. I think it's good to not have it because the compilation will fail if we add a new enum, which will remind us to implement proper logic for it.

Reply:
Is that the case though? I just tried adding another value to the enum, and everything still compiled just fine...

Reply:
Maybe on your system it does but our ci's will definitely pick it up they have generally more warnings turned on

Reply:
Yup, Duncan is right, not all compilers complain

Reply:
That works for me, thanks! I will update.